### ISSUE TITLE
Permission denial: starting intent

### ISSUE THREAD

**Issue by yergali28** [ricvalerio/foregroundappchecker#10]
Hi guys, I have this kind of problem in some devices with android version 5.1 - 5.1.1 ((

![2018-06-15 12 28 00](https://user-images.githubusercontent.com/33349723/41453439-9935754c-7097-11e8-9463-32e45e60aa7d.png)

**Comment by ricvalerio** [comment]
Hello,

Did you add the permissions in the manifest?

**Comment by yergali28** [comment]
Yes I added, everything works perfectly in other devices! In this kind of devices there is no activity in the settings!

**Comment by Duna** [comment]
#Olddevice or Amazon ones
